### PR TITLE
Update to containerd 1.5.1

### PR DESCRIPTION
**Pull request by adisky**
Update to containerd 1.5.1

Signed-off-by: Aditi Sharma <adi.sky17@gmail.com>

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -3,9 +3,22 @@ module github.com/containerd/nri
 go 1.14
 
 require (
+	github.com/Microsoft/go-winio v0.5.0 // indirect
 	// when updating containerd, adjust the replace rules accordingly
-	github.com/containerd/containerd v1.5.0-beta.3
+	github.com/containerd/containerd v1.5.0
```
Comment (mikebrow) [inline go.mod:8]: v1.5.1 is out..

**Review by mikebrow — COMMENTED** [review]
1.5.1 is out..

File: go.mod (hunk 1 above)
Comment (adisky) [reply go.mod]: Updated

**Comment by estesp** [comment]
Looks like `go.sum` needs to be updated again; can you `go mod tidy` and re-submit commit?

**Review by estesp — APPROVED** [review]
LGTM

**Review by mikebrow — APPROVED** [review]
LGTM

**Pull request merged**